### PR TITLE
Update webpage after 0.1.4

### PR DESCRIPTION
**Pull request by rok-cesnovar**
The webpage has not been updated since 0.1.2. 

Only ran `pkgdown::build_site()` and pushed the changes.

### CODE REVIEW

File: vignettes/rvar.Rmd [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "The Random Variable Datatype"
+title: "rvar: The Random Variable Datatype"
```
Comment (rok-cesnovar) [inline vignettes/rvar.Rmd:2]: This is the only other change. pkgdown complained that the title is not equal to VignetteIndexEntry which is:

`%\VignetteIndexEntry{rvar: The Random Variable Datatype}`

I decided to add rvar: to the title, but I am not sure what the intention was here.

cc: @mjskay

Comment (mjskay) [reply vignettes/rvar.Rmd:2]: Fine with me. I did make those different deliberately for a reason I can't currently recall, but if pkgdown is gonna complain making them agree is fine with me

Comment (rok-cesnovar) [reply vignettes/rvar.Rmd:2]: We can always ignore that warning or turn it off when building. I didnt know it was intentional.

Comment (mjskay) [reply vignettes/rvar.Rmd:2]: Nah, it's less complicated to make them match. I think I was copying a style from somewhere else, but I can't find it again and I can't recreate the rationale I had to make them not match.

**Comment by paul-buerkner** [comment]
Shall we merge this already?

**Comment by rok-cesnovar** [comment]
I am guessing it wont hurt. Having rvar docs online is mostly what got me to do this.

**Comment by paul-buerkner** [comment]
Thanks for taking care of this!

**Pull request merged**